pkgs/development/compilers/elm/packages/elm.nix

```diff
@@ -11,9 +11,8 @@ mkDerivation {
   version = "0.19.1";
```

Comment:
We should note that we're using a newer commit than 0.19.1

Reply:
directly in the version or just in a comment in code?

My understanding is that the official npm package is now also installing binaries from this commit as a version 0.19.1-6 https://www.npmjs.com/package/elm?activeTab=versions. Probably even official binaries are now updated https://github.com/elm/compiler/tree/master/installers/npm/packages even though the git tag is still pointing to commit from 2019 the binary artifacts are built from these newer commits.

Reply:
> directly in the version or just in a comment in code?

imo, I'd appreciate doing both (having the version string clearly indicate that it's not the upstream 0.19.1 tag, and also have a comment in here noting the current situation)